spec/features/user/user_can_see_github_repos_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe "As a logged in user, when i click on the github section" do
+  before(:each) do
+    @happy = User.create!(email: 'happymadison@example.com', first_name: 'Happy', last_name: 'Gilmore', password: '12345', token: ENV["github_api_token_k"])
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@happy)
```

Comment:
I had to change the environment variable name to be all underscores to abide by Travis's syntax for environment variables.... Will need to change on your local application.yml file